src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -85,9 +85,8 @@ public class NewAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
 
         [Parameter(
             Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "Active-Active feature flag")]
-        public bool ActiveActive { get; set; }
```

Comment:
Yes but these command lets are not yet released as in there are no customers who use these command lets yet. We will release the at Ignite. They were just part of last PS release but no customers using it yet.